sdk/spring/spring-cloud-azure-stream-binder-servicebus-core/src/main/java/com/azure/spring/cloud/stream/binder/servicebus/core/properties/ServiceBusConsumerProperties.java

```diff
@@ -5,12 +5,18 @@
 
 import com.azure.spring.messaging.servicebus.core.properties.ProcessorProperties;
 
+import java.time.Duration;
+
+import static com.azure.messaging.servicebus.implementation.ServiceBusConstants.MAX_DURATION;
+
 /**
  * Service Bus consumer properties.
  */
-public class ServiceBusConsumerProperties extends ProcessorProperties {
+public class ServiceBusConsumerProperties extends ProcessorProperties implements ServiceBusEntityOptionsProvider {
 
     private boolean requeueRejected;
+    private Long maxSizeInMegabytes = 1024L;
+    private Duration defaultMessageTimeToLive = MAX_DURATION;
```

Comment:
Should we add metadata for these two proeprties?

Reply:
The metadata definition is generally used for specific, known properties, AFAIK.